app/src/main/java/com/alphawallet/app/widget/NFTImageView.java

```diff
@@ -138,21 +140,22 @@ private void loadImage(String url, String backgroundColor, int corners) throws I
     {
         if (!Utils.stillAvailable(getContext())) return;
 
-        setWebViewHeight((int)getLayoutParams().width);
+        setWebViewHeight(getLayoutParams().height);
```

Comment:
I remember there used to be NFTs which that have extremely long and they covered the entire screen, and there were NFTs which were extremely wide as well. The conclusion was for me to use the code above to force it into a square, as a compromise. However I am not sure if that is still relevant because it has been a while.

Reply:
Thx for sharing this, I didn't find out what's the intention. I will revert it back.